packages/react/label/src/Label.tsx

```diff
@@ -81,39 +94,15 @@ Label.displayName = NAME;
 
 const useLabelContext = (element?: HTMLElement | null) => {
   const context = React.useContext(LabelContext);
+  const { onControlChange } = context || {};
```

Comment:
Should we use an empty obj as the initial context value instead?

Reply:
Provided a default context value instead